airflow/example_dags/plugins/event_listener.py

```diff
@@ -0,0 +1,77 @@
+from airflow.listeners import hookimpl
```

Comment:
`example_dags` feels a weird place

Reply:
hmm, the other plugin example `workday.py` was in the same folder. That's why I put it there. I could relocate all the plugin examples to a different file. Can I move it to example_plugins and put it on the same level as the example_dags folder?

Reply:
Fine for now, but something for the future

Reply:
Static checks are failing - https://github.com/apache/airflow/actions/runs/3547709746/jobs/5958172738

Reply:
I am checking this now.